frontend/pages/dashboard/my-posts.tsx

```diff
@@ -1,16 +1,51 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { NextPage } from 'next'
-
 import { withApollo } from '../../lib/apollo'
 import DashboardLayout from '../../components/Layouts/DashboardLayout'
+import TabToggle from '../../elements/TabToggle'
 
 const MyPostsPage: NextPage = () => {
+  const tabs = [
+    { key: 'published', text: 'Published' },
+    { key: 'drafts', text: 'Drafts' },
+  ]
+  const [activeKey, setActiveKey] = useState(tabs[0].key)
+
+  const handleToggle = (key: string): void => {
+    // TODO(nick): wire up query params so you can directly link to a tab
+    setActiveKey(key)
+  }
+
   return (
     <DashboardLayout>
-      <h1>My Posts</h1>
+      <TabToggle activeKey={activeKey} tabs={tabs} onToggle={handleToggle} />
+
+      <div className="posts-wrapper">
+        {activeKey === 'published' && <div>My Posts</div>}
```

Comment:
Does the text in the `<div>` need to be updated to "Published" as well? Is that what will show as the title?

Reply:
If it's the title I wonder if we could make it "Published Posts" and "Drafts

Reply:
These are just temporary divs that will be deleted and replaced with new components, probably something like `MyPosts.tsx` and `Drafts.tsx`. In the follow-up PR which adds these, I can add a heading/title above the posts 👍  `"My Published Posts"` and `"My Drafts"` sound good to me!

Reply:
Ah ok cool cool sounds good!